src/gsy_e/models/strategy/energy_parameters/energy_params_eb.py

```diff
@@ -77,31 +243,70 @@ def event_traded_energy(
         for time_slot, energy_value_kWh in trade_profile.items():
             self._state.decrement_energy_requirement(
                 purchased_energy_Wh=energy_value_kWh * 1000,
-                time_slot=time_slot,
+                time_slot=time_slot.set(tz=pendulum.UTC),
                 area_name=self._area.name)
 
 
-class ProductionStandardProfileEnergyParameters:
+class ProductionStandardProfileEnergyParameters(ForwardEnergyParams):
     """Energy parameters of the PV strategy for the forward markets."""
 
     def __init__(self, capacity_kW):
+        super().__init__()
         self.capacity_kW: float = capacity_kW
 
         self._state = PVState()
         self._area = None
         self._profile_generator: Optional[ForwardTradeProfileGenerator] = None
 
+    @property
+    def state(self) -> PVState:
+        """Retrieve the internal state of the asset."""
+        return self._state
+
     def serialize(self):
         """Return dict with the current energy parameter values."""
         return {
             "capacity_kW": self.capacity_kW
         }
 
+    @property
+    def capacity_kWh(self):
+        """Capacity of the PV in kWh."""
+        return convert_kW_to_kWh(self.capacity_kW, self._area.config.slot_length)
+
+    def get_available_energy_kWh(
+            self, market_slot: pendulum.DateTime, market_type: AvailableMarketTypes):
+        """Get the available offer energy of the PV."""
+        if market_type == AvailableMarketTypes.INTRADAY:
+            self._state.get_available_energy_kWh(market_slot)
```

Comment:
Nit:
```suggestion
            self.state.get_available_energy_kWh(market_slot)
```
(same in the rest of the module)

Reply:
Also replied above, fixed